addons/base_import/static/src/js/import_action.js

```diff
@@ -257,7 +261,6 @@ var DataImport = Widget.extend(ControlPanelMixin, {
 
         this.$el.removeClass('oe_import_preview oe_import_error');
         var import_toggle = false;
-        var file = this.$('input.oe_import_file')[0].files[0];
```

Comment:
Is this line breaking something?

Reply:
Removing it certainly breaks the following line.

Reply:
The file is retrieved l250 already, no need to re-get the file here

Reply:
but v11 so minimal diff